week-4/D-methods/exercise-5.js

```diff
@@ -9,35 +9,54 @@ only if the inserted amount is greater or equal than the price of the coffee!
 */
 
 var coffeeMachine = {
-    brand: "Super Coffee",
-    prices: {
-        cappuccino: 2.40,
-        blackCoffee: 1.50,
-        flatWhite: 3.00
-    },
-    insertedAmount: 0,
-    insertMoney: function(amount) {
-
-    },
-    getCoffee: function(coffee) {
-
+  brand: "Super Coffee",
+  prices: {
+    cappuccino: 2.4,
+    blackCoffee: 1.5,
+    flatWhite: 3.0
+  },
+  insertedAmount: 0,
+  insertMoney: function(amount) {
+      return this.insertedAmount = amount;
+      
+  },
+    getCoffee: function (coffee) {
+        if (coffee === "cappuccino" && this.insertedAmount >= this.prices.cappuccino) {
```

Comment:
Looks like this works (good!), but there's a way we can do this with many fewer lines of code, instead of a big if... else if... construction.  

Imagine there were 40 different types of product being vended, or 4000... We couldn't keep adding to the if... else if... without going insane.  
See if you can improve it.